holidays/countries/bhutan.py

```
@@ -167,6 +171,31 @@ def _populate_subdiv_15_public_holidays(self):
         self._add_thimphu_tshechu_day_three(name)
 
+    @property
+    def _bhutan_winter_solstice_date(self) -> tuple[int, int]:
+        """Extrapolation based on available dates; the exact rule is currently unclear."""
+        dates = {
+            2019: (JAN, 3),
+            2022: (JAN, 1),
+        }
+        if dt := dates.get(self._year):
+            return dt
+        else:
```

Ruff (0.15.14)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)